diff --git a/configure.in.in b/configure.in.in
index 2bfbab2..e309006 100644
--- a/configure.in.in
+++ b/configure.in.in
@@ -222,7 +222,7 @@ AM_CONDITIONAL(ENABLE_MONODOC, test "x$enable_monodoc" = "xyes")
 AM_CONDITIONAL(ENABLE_MSI, test "x$enable_msi" = "xyes")
 AM_CONDITIONAL(ENABLE_MONOGETOPTIONS, test "x$has_mono" = "xtrue")
 
-CSFLAGS="$DEBUG_FLAGS $CSDEFINES $WIN64DEFINES"
+CSFLAGS="-sdk:2 $DEBUG_FLAGS $CSDEFINES $WIN64DEFINES"
 
 PKG_CHECK_MODULES(GLIB_2_31,
   glib-2.0 >= 2.31,
